app/src/main/java/io/pslab/communication/ScienceLab.java

```diff
@@ -1879,6 +1879,23 @@ public void startOneChannelLABackup(Integer trigger, String channel, Integer max
         }
     }
 
+    /**
+     * Start logging timestamps of rising/falling edges on ID1.
+     * @param channel ['ID1','ID2','ID3','ID4','SEN','EXT','CNTR']
+     * @param channelMode acquisition mode
+                        default value: 1(EVERY_EDGE)
+                        - EVERY_SIXTEENTH_RISING_EDGE = 5
+                        - EVERY_FOURTH_RISING_EDGE    = 4
+                        - EVERY_RISING_EDGE           = 3
+                        - EVERY_FALLING_EDGE          = 2
+                        - EVERY_EDGE                  = 1
+                        - DISABLED                    = 0
+     * @param triggerChannel ['ID1','ID2','ID3','ID4','SEN','EXT','CNTR']
+     * @param triggerMode    1=Falling edge
+                             0=Rising Edge
+                            -1=Disable Trigger
+     */
+
```

Comment:
Let's remove these extra line breaks everywhere in the PR and make it looks uniform